src/test/ScrollStateBridge.t.sol

```diff
@@ -92,18 +92,10 @@ contract ScrollStateBridgeTest is PRBTest, StdCheats {
 
     function setUp() public {
         /// @notice Create a fork of the Ethereum mainnet
-        mainnetFork = vm.createFork(MAINNET_RPC_URL);
+        mainnetFork = vm.createSelectFork(SEPOLIA_RPC_URL);
 
-        vm.selectFork(mainnetFork);
-        /// @notice Roll the fork to a block where both Optimim's and Base's crossDomainMessenger contract is deployed
-        /// @notice and the Base crossDomainMessenger ResolvedDelegateProxy target address is initialized
-        vm.rollFork(17711915);
-
-        if (block.chainid == 1) {
-            opCrossDomainMessengerAddress = address(0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1);
-        } else {
-            revert invalidCrossDomainMessengerFork();
-        }
+        opCrossDomainMessengerAddress = address(0x6774Bcbd5ceCeF1336b5300fb5186a12DDD8b367);
```

Comment:
@Jemiiah Shouldn't this be Scroll and not Op (optimism)?

Reply:
yes @dragan2234 it's scroll address I forgot to rename it